eng/tox/import_all.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# This script is used to verify package dependency by importing all modules
+import os
+
+#get current package name
+current_dir = os.path.basename(os.getcwd())
+package_name = current_dir.replace('-','.')
```

Comment:
One change. Please pass the `{toxinidir}` as an argument to this package, then parse that. I don't want to rely on `os.getcwd()`

Reply:
You can see how I do this in any of the other `run_pylint` or `run_sphinx` scripts.

Reply:
I have made this changes and new PR has completed checks.